tools/biotransformer/biotransformer.xml

```diff
@@ -0,0 +1,131 @@
+    </data>
+    <data format="csv" name="output_file2" label="BioTransformer - filtered CSV output.">
+	  <filter>input['type_selector'] == 'csv'</filter>
+    </data>
+
+    <data format="csv" name="output_file3" label="BioTransformer - super-filtered CSV output.">
+	  <filter>input['type_selector'] == 'csv'</filter>
+    </data>
+  </outputs>
+  <help>
+This is version 1.1.0 of BioTransformer. BioTransformer is a software tool that predicts small molecule metabolism in mammals, their gut micr
+obiota, as well as the soil/aquatic microbiota. BioTransformer also assists scientists in metabolite identification, based on the metabolism
+prediction.
+
+BioTransformer is offered to the public as a freely acessible software package under the GNU License GPL v2.1.
+
+Users are free to copy and redistribute the material in any medium or format. Moreover, they could modify, and build upon the material unfer
+the condition that they must give appropriate credit, provide links to the license, and indicate if changes were made. Furthermore, the above
+ copyright notice and this permission notice must be included. Use and re-distribution of the these resources, in whole or in part, for comme
+rcial purposes requires explicit permission of the authors. We ask that all users of the BioTransformer software tool, the BioTransformer web
+ server, or BioTransformerDB to cite the BioTransformer reference in any resulting publications, and to acknowledge the authors.
+
+Parameters explanation:
+
+Input. User can chose from SDF, SMILES and CSV. CSV expects file with one moleluce in SMILES format per line. If CSV is selected, OUTPUT MUST
+ BE CSV AS WELL!
+
+The type of description: Type of biotransformer - EC-based (ecbased), CYP450 (cyp450), Phase II (phaseII), Human gut microbial (hgut), human
+super transformer* (superbio, or allHuman), Environmental microbial (envimicro).
+
+The number of steps for the prediction. This option will be useed for the EC-based, CYP450, Phase II, and Environmental microbial biotransfor
+mers. The default value is 1.
+
+Mass tolerance for metabolite identification (default is 0.01).
+
+Output of BioTransformer with CSV as an input are 3 CSV files. One without any filtering, second with filtered duplicates based on 6 columns
+(InChI, InChIKey, Synonyms, Molecular formula, Major Isotope Mass, AlogP) and third with filtered duplicates based on 3 columns (Molecular fo
+rmula, Major Isotope Mass, AlogP).
+
+Output of BioTransformer with SMILES as an input is 1 CSV file.
+
+(* ) While the 'superbio' option runs a set number of transformation steps in a pre-defined order (e.g. deconjugation first, then Oxidation/r
+eduction, etc.), the 'allHuman' option predicts all possible metabolites from any applicable reaction(Oxidation, reduction, (de-)conjugation)
+ at each step.
+
+Cite: Djoumbou-Feunang Y, Fiamoncini J, de la Fuente AG, Manach C, Greiner R, and Wishart DS; BioTransformer: A Comprehensive Computational T
```

Comment:
We could use the `<citation>` galaxy tool element here to render the citation in bibtex. See https://docs.galaxyproject.org/en/master/dev/schema.html#tool-citations

Reply:
Added citation tag as suggested.